table/table.go

```diff
@@ -173,7 +171,20 @@ func OpenTable(fd *os.File, mode options.FileLoadingMode, cksum []byte) (*Table,
 
 	switch mode {
 	case options.LoadToRAM:
-		// No need to do anything. t.mmap is already filled.
+		if _, err := t.fd.Seek(0, io.SeekStart); err != nil {
+			return nil, err
+		}
+		t.mmap = make([]byte, t.tableSize)
+		n, err := t.fd.Read(t.mmap)
+		if err != nil {
+			// It's OK to ignore fd.Close() error because we have only read from the file.
+			_ = fd.Close()
```

Comment:
t.fd?

Reply:
Yes. Changed in https://github.com/dgraph-io/badger/pull/816/commits/fc950be2fe51ca589fb19c77e33bf10cafb9c1bd